Lib/test/test_fnmatch.py

```diff
@@ -72,7 +72,7 @@ def test_fnmatchcase(self):
 
     def test_bytes(self):
         self.check_match(b'test', b'te*')
-        self.check_match(b'test\xff', b'te*\xff')
+        self.check_match(b'test\xe1\x88\xb4', b'te*\xe1\x88\xb4')
```

Comment:
Old implementation of `ntpath.normcase` support invalid encoded  binary strings as input as here. I think real codes do not have usecases like here, so just changed the test case here. I have no idea about is this suitable. cc @eryksun